core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -136,138 +141,537 @@ class DescribeConsumerGroupTest extends KafkaServerTestHarness {
   }
 
   @Test
-  def testDescribeNonExistingGroupWithNewConsumer() {
+  def testDescribeNonExistingGroup() {
+    TestUtils.createOffsetsTopic(zkUtils, servers)
+    val missingGroup = "missing.group"
+
+    for (describeType <- describeTypes) {
+      // run one consumer in the group consuming from a single-partition topic
+      addConsumerGroupExecutor(new ConsumerGroupExecutor(brokerList, 1, group, topic))
```

Comment:
I guess you're following a pre-existing pattern, but running a separate group seems like overkill for this test case.

Reply:
I was more wondering if it needed to be in the test at all since we don't actually use it.